src/index.html

```diff
@@ -17,11 +17,452 @@
     />
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="./styles/index.scss"
     />
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__logo">
+        <a href="#">
+          <img
+            src="../src/images/logo.png"
+            alt="logo"
+            class="header__logo"
```

Comment:
The `img` element has a class `header__logo`, which is the same as the parent `div`. It's better to use a different class name for the `img` to avoid confusion and potential styling conflicts.